### PR TITLE
chore(docs): migrate react-notifications examples to storybook

### PR DESCRIPTION
**Pull request by hzhu**
## Description

This PR migrates `react-notifications` examples to Storybook.

## Checklist

- [x] :arrow_left: renders as expected with reversed (RTL) direction
- [x] :metal: renders as expected with Bedrock CSS (`?bedrock`)
- [x] :wheelchair: analyzed via [axe](https://www.deque.com/axe/) and evaluated using VoiceOver
- [x] :memo: tested in Chrome, Firefox, Safari, Edge, and IE11

### CODE REVIEW

File: packages/notifications/stories/3-Notification.stories.tsx [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+Default.args = {
+  type: 'success'
+};
+
+Default.argTypes = {
+  isRegular: {
+    control: 'boolean'
+  },
+  type: {
+    control: { type: 'select', options: ['success', 'warning', 'error', 'info'] }
```
Comment (jzempel) [inline packages/notifications/stories/3-Notification.stories.tsx:42]: Please add support for the default `undefined` case.

File: packages/notifications/stories/4-Well.stories.tsx [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+    <Row>
+      <Col>
+        <Well isFloating={isFloating} isRecessed={isRecessed}>
+          <Title>Nori Grape Beet</Title>
+          <Paragraph>
+            Nori grape silver beet broccoli kombu beet greens fava bean potato quandong celery.
+            Bunya nuts black-eyed pea prairie turnip leek lentil turnip greens parsnip. Sea lettuce
+            lettuce water chestnut eggplant winter purslane fennel azuki bean earthnut pea sierra
+            leone bologi leek soko chicory celtuce parsley jícama salsify black-eyed pea quandong.
+          </Paragraph>
```
Comment (jzempel) [inline packages/notifications/stories/4-Well.stories.tsx:29]: Can we see controls for the regular weight title and multi-line paragraph here?

File: packages/notifications/stories/3-Notification.stories.tsx [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+} as Meta;
+
+export const Default: Story = ({ type, isRegular }) => (
+  <Grid>
+    <Row>
+      <Col>
+        <Notification type={type}>
+          <Title isRegular={isRegular}>Turnip greens</Title>
+          Turnip greens yarrow ricebean rutabaga endive cauliflower sea lettuce kohlrabi amaranth
+          water
```
Comment (jzempel) [inline packages/notifications/stories/3-Notification.stories.tsx:25]: Can we see a control for a multi-line paragraph here?

**Review by jzempel — CHANGES_REQUESTED** [review]

File: packages/notifications/stories/4-Well.stories.tsx (hunk 2 above)
Comment (hzhu) [reply packages/notifications/stories/4-Well.stories.tsx]: Yeah a control could be added for that.

**Review by jzempel — APPROVED** [review]

**Pull request merged**